packages/odyssey-react-mui/src/Typography.test.tsx

```diff
@@ -23,4 +23,54 @@ describe("Typography", () => {
 
     expect(screen.getByLabelText("overline")).toBeVisible();
   });
+
+  test("renders with role", () => {
+    render(
+      <Typography ariaLabel="heading" variant="h1" role="presentation">
+        Heading test
+      </Typography>,
+    );
+
+    const element = screen.getByLabelText("heading");
+    expect(element).toBeVisible();
+    expect(element).toHaveAttribute("role", "presentation");
+  });
+
+  test("does not render role attribute when undefined", () => {
+    render(
+      <Typography ariaLabel="heading" variant="h1" role={undefined}>
+        Heading test
+      </Typography>,
+    );
```

Comment:
```suggestion
  test("does not render role attribute when undefined", () => {
    render(
      <Typography ariaLabel="heading" variant="h1">
        Heading test
      </Typography>,
    );
```

Is there a reason we don't just remove the `role` attribute altogether? It's optional right?

I don't want to specifically pass `undefined` in our test as our users are simply not passing anything; therefore, this is testing a use case that no one has, and if it breaks when you don't pass a value to `role`, we won't know.